src/Extensions.Hosting.AsyncInitialization/IAsyncInitializer.cs

```diff
@@ -11,6 +12,6 @@ public interface IAsyncInitializer
         /// Performs async initialization.
         /// </summary>
         /// <returns>A task that represents the initialization completion.</returns>
```

Comment:
Why did you remove the doc for the `cancellationToken` parameter?
The only thing I wanted you to change here was the `.` that you had removed in the summary, everything else was fine 😉
I had added the comment on that line, so I thought it was clear, but maybe I should have been more specific.